frontend/src/components/ErrorBanner.jsx

```
@@ -0,0 +1,23 @@
+
+const ErrorBanner = ({ error, onRetry, className }) => {
+  if (!error) return null;
+  
+  return (
+    <div className={`${styles.errorBanner} ${className || ''}`}>
+      <p>{error}</p>
+      {onRetry && (
+        <button 
+          className={styles.retryButton}
+          onClick={onRetry}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)